src/VirtoCommerce.CatalogModule.BulkActions/VirtoCommerce.CatalogModule.BulkActions.csproj

```diff
@@ -22,8 +22,8 @@
 
     <ItemGroup>
         <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
-        <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.8" />
-        <PackageReference Include="VirtoCommerce.BulkActionsModule.Core" Version="3.1.0" />
+        <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.18" />
```

Comment:
current Platform references 
<PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.13" />
and module can't reference higher version then Platform does. Sync them.